src/braket/aws/aws_quantum_task_batch.py

```diff
@@ -331,7 +336,9 @@ def results(
         fail_unsuccessful: bool = False,
         max_retries: int = MAX_RETRIES,
         use_cached_value: bool = True,
-    ) -> list[AwsQuantumTask]:
+    ) -> list[
+        GateModelQuantumTaskResult | AnnealingQuantumTaskResult | PhotonicModelQuantumTaskResult
```

Comment:
Should this also include AHS result types?

Reply:
Updated